artifacts/rhcos/rhcos.go

```diff
@@ -18,10 +19,17 @@ type rhcos struct {
 	Compression string
 }
 
+var description string = `RHCOS images for KubeVirt.
+<br />
+<br />
+Visit [https://docs.openshift.com/container-platform/4.9/architecture/architecture-rhcos.html) to learn more about Red Hat Enterprise Linux CoreOS.`
```

Comment:
bummer that this has the `4.9` version in the link. Would it be worth making that dynamic?

Reply:
Fixed. Turns out that if you replace `4.9` with `latest` a redirect to the latest version will take place :)